asteroid/masknn/tac.py

```diff
@@ -52,6 +52,8 @@ def forward(self, x, valid_mics):
         """
         # Input is 5D because it is multi-channel DPRNN. DPRNN single channel is 4D.
         batch_size, nmics, channels, chunk_size, n_chunks = x.size()
+        if valid_mics is None:
+            valid_mics = torch.LongTensor([nmics] * batch_size)
```

Comment:
If we don't have a default value, we cannot use it in the inference CLI. 

If there is no `valid_mics`, we consider all recordings to have the max number of mics. 
@popcornell , can you validate that this is ok?

Reply:
Yes this is a sensible default

Reply:
Cool thanks.